### DISCUSSION TITLE
Master -> main, but no main

### DISCUSSION THREAD

**Discussion by mannyveloso** [JamesIves/github-pages-deploy-action#677] [Q&A]
The master branch was changed to main (675), but main isn't found. What's the equivalent of what used to be master?

**Comment by JamesIves** [comment]
[Please follow the README to setup the action](https://github.com/JamesIves/github-pages-deploy-action), the master branch has been deprecated for a long time now and was completely removed this week. There is no equivalent called main. 

In general it's not a good idea to point directly to a release branch with an action, instead point to a tagged version.

**Reply by mannyveloso** [reply]
Will do.

**Reply by mannyveloso** [reply]
So maybe you didn't understand the question. Which one of the releases was closest to master?

**Reply by JamesIves** [reply]
Master was based on version 2, I strongly recommended using the most recent version to prevent things from randomly breaking.
